R/shiny_devmode.R

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Add a comment here about how we could avoid duplication here if we ever decide to import shiny.